woocommerce/woocommerce-android
# [Activate] Remove the magic link sent experiment

This PR removes the magic link experiment sent experiment after it's been concluded that the `improved` variant (which shows the screen below after requesting a magic link) performs the best.

![image](https://user-images.githubusercontent.com/1522856/194086928-201d38d2-dea4-451a-aec4-a09bce79788d.png)

**To test:**
1. Start the app in a logged out state
2. Enter your email address that is eligible for a magic link
3. Request the magic link
4. Verify the screen with the envelop is displayed
5. Tap on the `Use password to sign in` button
6. Verify the password screen is shown

**Note:** Please wait until #7486 is merged before merging this PR.
